app/views/users/index.html.erb

```diff
@@ -1,9 +1,33 @@
-<h1>Users</h1>
-
-<ul>
-<% @users.each do |user| %>
-   <li>
-   <%= user.name %>
-   </li>
-<% end %>
-</ul>
\ No newline at end of file
+   <html>
+  <head>
+    <meta http-equiv = "Content-Type" content = "text/html; charset = iso-8859-1" />
+    <meta http-equiv = "Content-Language" content = "en-us" />
+    <title>Library Info System</title>
+    <%= stylesheet_link_tag "style" %>
+  </head>
+  <body>
+    <h1>Users</h1>
+    <table>
+      <tr>
+        <th>
+          <h1>ID</h1>
+        </th>
+        <th>
+          <h1>Users</h1>
+        </th>
+      </tr>
+      <% @users.each do |user| %>
+        <tr>
+          <td>
+            <h2><%= user.id %></h2>
+          </td>
+          <td>
+            <h2><%= user.name %><br>
+            </h2>
+            Number of posts: <%= user.posts_counter %>
+          </td>
+        </tr>
+      <% end %>
+    </table>
+  </body>
+</html>
```

Comment:
- [ ] As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Note%3A%20Analyze%20the%20wireframes%20deeply%20and%20try%20to%20identify%20all%20the%20necessary%20elements%20and%20how%20a%20user%20would%20navigate%20from%20one%20page%20to%20another) requirement. Please make sure that the user can navigate between the pages to access all of the specified routes. For example, in the `Users#index` view, the user can click on one of the users' cards to access the `Users#show` view of the clicked card.